changelogs/fragments/7811.yml

```diff
@@ -0,0 +1,2 @@
+security:
```

Comment:
not related to this PR. 
in 2.14 we proved we could generate release note without changelog file. the maintainers should consider deprecate the needs of this file and simplify the contribution process

Reply:
This is the auto-generated one.